trunk/SUAVE/Methods/Center_of_Gravity/compute_aircraft_center_of_gravity.py

```diff
@@ -2,6 +2,9 @@
 # 
```

Comment:
We should consider splitting these into multiple files instead of having low level if statements.